Rest/Controller/ContentController.php

```diff
@@ -0,0 +1,199 @@
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException If the user has no access to read content and in case of un-published content: read versions
+     */
+    public function getContent($contentId)
+    {
+        $contentIds = explode(',', $contentId);
+        $content = $this->prepareContent($contentIds);
+
+        return new ContentDataValue($content);
+    }
+
+    /**
+     * Prepare nested content array.
+     *
+     * @param $contentIds
+     * @param int $nesting Maximum nesting level for container content.
+     *
+     * @return array
+     */
+    protected function prepareContent($contentIds)
+    {
+        $contentService = $this->repository->getContentService();
+        $locationService = $this->repository->getLocationService();
+        $contentTypeService = $this->repository->getContentTypeService();
+
+        $requestLanguage = $this->request->get('lang');
+        $requestedFields = $this->request->get('fields');
+
+        $content = [];
+
+        foreach ($contentIds as $contentId) {
+            $contentValue = $contentService->loadContent((int) $contentId, [$requestLanguage]);
+            $contentType = $contentTypeService->loadContentType($contentValue->contentInfo->contentTypeId);
+            $location = $locationService->loadLocation($contentValue->contentInfo->mainLocationId);
+            $user = $this->repository->getUserService()->loadUser($contentValue->contentInfo->ownerId);
+            $language = (is_null($requestLanguage)) ? $contentType->mainLanguageCode : $requestLanguage;
+
+            $content[$contentId] = [
+                'contentId' => (int) $contentId,
+                'contentTypeId' => $contentType->id,
+                'identifier' => $contentType->identifier,
+                'language' => $language,
+                'publishedDate' => $contentValue->contentInfo->publishedDate->format('c'),
+                'author' => $user->getFieldValue('first_name').' '.$user->getFieldValue('lasst_name'),
+                'uri' => $this->generator->generate($location, [], false),
+                'mainLocation' => [
+                    'href' => '/api/ezp/v2/content/locations'.$location->pathString,
+                ],
+                'locations' => [
+                    'href' => '/api/ezp/v2/content/objects/'.$contentId.'/locations',
+                ],
+                'categoryPath' => $location->pathString,
+            ];
+
+            $fields = $this->prepareFields($contentType, $requestedFields);
+
+            if (!is_null($fields)) {
+                foreach ($fields as $field) {
+                    $fieldValue = $contentValue->getFieldValue($field, $language);
+
+                    if (is_null($fieldValue)) {
+                        continue;
+                    } elseif (key($fieldValue) == 'destinationContentId') {
```

Comment:
Looks for _destinationContentId_ key. If found it should return related image uri.